packages/embedded-wallet/package.json

```diff
@@ -33,7 +33,7 @@
     }
   },
   "lint-staged": {
-    "src/**/*.tsx?": "eslint"
+    "src/**/*.{ts,tsx}": "eslint"
```

Comment:
Why does `tsx?` not work?

Reply:
`lint-staged` only implements a subset of `regex` patterns, it seems. https://github.com/statechannels/monorepo/blame/199a552da29e6824b77d6b047adc73eccb8f48ef/packages/wallet/src/redux/adjudicator-state/reducer.ts#L47:L47

Reply:
The docs there indicate that it uses [micromatch](https://github.com/micromatch/micromatch), which claims to support the `*` and `?` wildcards.